### PR TITLE
[BYOC][Verilator] Skip mobilenet test if Verilator is not available

### PR DESCRIPTION
**Pull request by vegaluisjose**
@leandron @tmoreau89

### CODE REVIEW

**Review by tmoreau89 — APPROVED** [review]
Approved. @leandron thank you for flagging the issue.

File: tests/python/contrib/test_verilator/test_mobilenet.py [hunk 1]
```diff
@@ -220,6 +221,8 @@ def tmobilenet(lanes):
     lanes : Int
         The number of vector lanes.
     """
+    if skip_test():
```
Comment (leandron) [inline tests/python/contrib/test_verilator/test_mobilenet.py:224]: One small note about this approach, when compared to the `skipIf` is that if you have no Verilator and you run these tests, once you reach the `return`, you'll see `PASSED` in the report, which might give the illusion that the test ran and passed.

The `skipIf` will be a bit more precise and report `SKIPPED`, so it is clear that the test was not run.

**Review by leandron — COMMENTED** [review]
Thanks @vegaluisjose, just a small note on the approach to skip the test.

**Review by leandron — APPROVED** [review]
LGTM waiting CI, then feel free to merge it.

Thanks @vegaluisjose!

**Comment by leandron** [comment]
Merged now. thanks @vegaluisjose @tmoreau89 !

**Pull request merged**